src/Analysis/GWASResultsV2/Views/Home/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { useContext } from 'react';
+import { Spin } from 'antd';
+import HomeTable from './HomeTable/HomeTable';
+import SharedContext from '../../Utils/SharedContext';
+
+const Home = () => {
+  const { tableData } = useContext(SharedContext);
+  return (
+    <div>{tableData && tableData.length > 0 ? <HomeTable /> : <Spin />}</div>
```

Comment:
so if `tableData.length == 0` we show a spinner? Not sure if we want that. Maybe just show an empty table with some "no data" indication? Something like this:

![Screenshot 2023-03-08 at 17 46 35](https://user-images.githubusercontent.com/2900303/223775919-d8e0fc5c-7c6b-4507-9f65-2e4c1909ddfc.png)

Reply:
I removed the check for tableData.length > 0 for now, in ticket VADC 465 or 466 the UX for no data will be addressed.